docs/pages/enterprise/hsm.mdx

```diff
@@ -40,166 +113,86 @@ auth_service:
+      # slot_number should be set to 0 for YubiHSM2
       slot_number: 0
-      pin: "<key_id><password>" # eg "ce9bhunter2"
+      # pin should be the (hex) slot of your authentication key,
+      # concatenated with the password
+      pin: "53bfhunter22"
       # pin_path can optionally be used to read the pin from a file
       # pin_path: /path/to/pin_file
 ```
-6. Set the `YUBIHSM_PKCS11_CONF` environment variable to the path of your
-   `yubihsm-pkcs11.conf` file and start Teleport.
-
-## AWS CloudHSM
-
-1. Create a VPC in your desired region, with private subnets in each desired Availability Zone.
-
-2. Create a CloudHSM cluster in your VPC and subnets.
-
-3. Initialize the cluster.
-   - Create an HSM in your desired AZ, and wait for it to be created.
-   - Download the CSR (Certificate Signing Request).
-   - Sign the CSR with an appropriate CA.
-   - Upload the signed certificate and the issuing CA certificate to complete
-      initialization of the HSM
-
-4. Create an EC2 instance that can reach your HSM VPC subnet.
+  </TabItem>
+</Tabs>
 
-5. Add the security group named after the CloudHSM cluster to allow incoming
-   traffic from the cluster's security group on ports 2223-2225.
+## Step 3/3. (Re)start Teleport Auth
 
-6. Copy the issuing certificate from step 3 to `/opt/cloudhsm/etc/customerCA.crt`
+If this is a new auth server starting a brand new cluster, HSM keys will be
+automatically generated at startup and no further steps are required. Check the
+teleport logs for `Creating new HSM key pair` to confirm that the feature is
+working. You can also check that keys were created in your HSM.
 
-7. Install the AWS CloudHSM Client SDK 5
-[https://docs.aws.amazon.com/cloudhsm/latest/userguide/pkcs11-library-install.html]
-```bash
-wget https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL6/cloudhsm-pkcs11-latest.el6.x86_64.rpm
-sudo yum install ./cloudhsm-pkcs11-latest.el6.x86_64.rpm
-```
+If you are connecting an HSM to an existing Teleport cluster, restart the auth
+server for the configuration changes to take effect. New CA keys will
+automatically be generated in the HSM. For these keys to be trusted by the rest
+of the cluster you will need to permorm a CA rotation, see
+[Next Steps](#certificate-rotation-with-hsm). The auth server will not perform
+any signing operations until the rotation has started. In an HA cluster you
+should add the HSM to the auth configuration one server at a time, and do not
+route any traffic to the auth server where the HSM is currently being added.
 
-8. Activate the cluster.
-```bash
-sudo /opt/cloudhsm/bin/configure --cmu <IP address>
-/opt/cloudhsm/bin/cloudhsm_mgmt_util /opt/cloudhsm/etc/cloudhsm_mgmt_util.cfg
-> listUsers
-> loginHSM PRECO admin password
-> changePswd PRECO admin <NewPassword>
-> listUsers
-# user `admin` should change from PRECO to CO(Crypto Officer)
-```
+## Next Steps
+### Certificate Rotation with HSM
```

Comment:
@nklaassen, sorry, I misunderstood. After reading this guide again, it seems like CA rotation is a necessary step to kick off HSM. Let's make it Step 4/4 then. The section `Next steps` is usually for optional steps, and this one seems like mandatory.

Reply:
Right, unless you are starting a brand new auth server and cluster it is mandatory, I will make this step 4